src/server.cpp

```diff
@@ -479,6 +470,7 @@ void Server::publishStatistics() const
     msg.data = s.str();
 
     pub_stats_.publish(msg);
+    updater_.force_update();
```

Comment:
Why is this neccessary? I would think this is a read-only function, hence why it was originally constant. Why can't we maintain that?

Reply:
`force_update` is a non-const function. I can't change that.